sos/report/plugins/ovn_central.py

```diff
@@ -161,8 +157,9 @@ def setup(self):
         cmds += sbctl_cmds
 
         # If OVN is containerized, we need to run the above commands inside
-        # the container.
-
+        # the container. Removing duplicates (in case there are) to avoid
+        # failing on collecting output to file on container running commands
+        cmds = list(set(cmds))
```

Comment:
ACK to this as a workaround. While `sos` should be clever enough to prevent raising the exception you wrote about. Could you please open a new issue with an artificial reproducer "run sos in container with plugin that collects two identical commands, that raises exception X", to let us fix that later on?

Reply:
done! https://github.com/sosreport/sos/issues/3131